streamalert/scheduled_queries/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+
+StreamQuery base module
+"""
+__version__ = '1.0.0'
```

Comment:
I don't know if we need explicit versioning for this submodule. thoughts?

Reply:
It's something I was going to remove, but see above comment on copy pasta

Reply:
Removed